packages/dm-core-plugins/src/explorer/components/dialogs/NewEntityDialog.tsx

```diff
@@ -14,6 +16,7 @@ import {
   STANDARD_DIALOG_WIDTH,
 } from '../context-menu/NodeRightClickMenu'
 
+const defaultName = 'new_entity'
```

Comment:
Could add a random suffix to avoid conflicts

Reply:
removed it completely